cmd/storage-rest-client.go

```diff
@@ -50,9 +50,18 @@ func isNetworkError(err error) bool {
 		return xnet.IsNetworkOrHostDown(nerr.Err, false)
 	}
 
+	switch err.Error() {
```

Comment:
Maybe `strings.Contains(err.Error(), "server closed idle connection")` - http likes to wrap errors.

Reply:
> Maybe strings.Contains(err.Error(), "server closed idle connection") - http likes to wrap errors.

Yes, done.. this is not a hot path anyway